app/containers/message/Content.tsx

```diff
@@ -72,7 +71,7 @@ const Content = React.memo(
 		}
 
 		// If this is a encrypted message and is not a preview
-		if (props.type === E2E_MESSAGE_TYPE && !isPreview) {
+		if (props.type === E2E_MESSAGE_TYPE && !isPreview && !props.isHeader) {
```

Comment:
I was wondering if we should move this block of code to: `app/containers/message/Message.tsx` line 108. To maintain all icon components together.